src/main/kotlin/com/exactpro/th2/http/client/Main.kt

```diff
@@ -214,7 +215,8 @@ data class Settings(
     val host: String,
     val port: Int = if (https) 443 else 80,
     val readTimeout: Int = 5000,
-    val socketCapacity: Int = 5,
+    val maxParallelRequests: Int = 5,
+    val maxThreadCount: Int = 5,
```

Comment:
Why do we need a separate parameter for this? `maxParallelRequests` already says how much threads we need to execute requests

Reply:
Redunant code, forgot to remove